lib/dune

```diff
@@ -3,27 +3,20 @@
  (public_name io-page)
  (modules Io_page)
  (libraries cstruct bigarray-compat)
+ (c_names stub_alloc_pages stub_get_addr)
  (js_of_ocaml (javascript_files io-page.js))
  (wrapped false))
 
 (library
- (name io_page_xen)
- (public_name io-page-xen)
+ (name io_page_unix)
+ (public_name io-page-unix)
+ (modules)
  (libraries io-page)
- (modules Io_page_xen_dummy)
- (c_names stub_alloc_pages_xen stub_get_addr_xen)
- (c_flags (:include c_flags_xen.sexp))
- (wrapped false))
-
-(rule
- (targets c_flags_xen.sexp)
- (deps config/discover.exe)
- (action (run %{deps})))
+)
 
 (library
- (name io_page_unix)
- (public_name io-page-unix)
- (libraries io-page bigarray)
- (modules Io_page_unix_dummy)
- (c_names stub_alloc_pages stub_get_addr)
- (wrapped false))
+ (name io_page_xen)
+ (public_name io-page-xen)
+ (modules)
+ (libraries io-page)
+)
```

Comment:
very minor but `dune fmt` will format these by putting `))` on the same line :-)